### PR TITLE
release 0.4.2 (#367)

### LINKED ISSUE #367: release 0.4.2

### PR DESCRIPTION
**Pull request by adietish**
fixes #367

### CODE REVIEW

File: src/main/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -11,6 +11,10 @@
     <change-notes>
         <![CDATA[
         <p></p>
+        <h3>0.4.2</h3>
+        <ul>
+            <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/365">IllegalStateException when binary plugin is installed in EAP 2021.3.3"</a></li>
```
[the remark this answers is not in the document]
Comment (adietish) [reply src/main/resources/META-INF/plugin.xml:16]: EAP is wrong now, 2021.3.3 was released: https://youtrack.jetbrains.com/articles/IDEA-A-111/IntelliJ-IDEA-2021.3.3-(213.7172.25-build)-Release-Notes?utm_source=product&utm_medium=link&utm_campaign=TBA
Correcting the wording.

[the remark this answers is not in the document]
Comment (adietish) [reply src/main/resources/META-INF/plugin.xml:16]: @jeffmaury: corrected, please re-review

[the remark this answers is not in the document]
Comment (jeffmaury) [reply src/main/resources/META-INF/plugin.xml:16]: Seems commit is missing as I'm still seeing EAP

[the remark this answers is not in the document]
Comment (adietish) [reply src/main/resources/META-INF/plugin.xml:16]: I suspect a github caching issue. Altered commit was pushed force:
image
image

[the remark this answers is not in the document]
Comment (adietish) [reply src/main/resources/META-INF/plugin.xml:16]: issue doesnt seem to go away. Creating a new PR

**Pull request closed without merging**